### ISSUE TITLE
Unable to start neo4j 3.3.4 with graphaware nlp plugins

### ISSUE THREAD

**Issue by smartinsightsfromdata** [graphaware/neo4j-nlp#83]
My error seems to be slightly different than the one reported https://github.com/graphaware/neo4j-nlp/issues/81 hence I've created another issue.

My configuration:
```
OSX 10.13.4
neo4j 3.3.4
java jdk jdk-8.0_172.jdk
```

graphaware components installed in plugin folder:
```
graphaware-nlp-3.4.0.52.8.jar
graphaware-server-enterprise-all-3.4.0.52.jar
nlp-stanfordnlp-3.4.0.52.8.jar
```

Changes to the conf file:
```
# dbms.directories.import=import
dbms.security.auth_enabled=false
dbms.memory.heap.initial_size=2512m
dbms.memory.heap.max_size=4512m  # should be sufficient to start?

dbms.unmanaged_extension_classes=com.graphaware.server=/graphaware
com.graphaware.runtime.enabled=true
com.graphaware.module.NLP.1=com.graphaware.nlp.module.NLPBootstrapper
dbms.security.procedures.whitelist=ga.nlp.*
```
Before starting the file with graphaware plugins I delete the database, to start with a clean slare.

This is the error:
```
2018-05-24 13:04:36.466+0000 INFO  ======== Neo4j 3.3.4 ========
2018-05-24 13:04:36.664+0000 INFO  Starting...
2018-05-24 13:04:38.616+0000 INFO  Bolt enabled on 0.0.0.0:7687.
2018-05-24 13:04:40.533+0000 INFO [c.g.r.b.RuntimeKernelExtension] GraphAware Runtime enabled, bootstrapping...
2018-05-24 13:04:40.569+0000 ERROR Failed to start Neo4j: Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@14e2e1c3' was successfully initialized, but failed to start. Please see the attached cause exception "org.neo4j.kernel.impl.core.EmbeddedProxySPI". Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@14e2e1c3' was successfully initialized, but failed to start. Please see the attached cause exception "org.neo4j.kernel.impl.core.EmbeddedProxySPI".
org.neo4j.server.ServerStartupException: Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@14e2e1c3' was successfully initialized, but failed to start. Please see the attached cause exception "org.neo4j.kernel.impl.core.EmbeddedProxySPI".
	at org.neo4j.server.exception.ServerStartupErrors.translateToServerStartupError(ServerStartupErrors.java:68)
	at org.neo4j.server.AbstractNeoServer.start(AbstractNeoServer.java:220)
	at org.neo4j.server.ServerBootstrapper.start(ServerBootstrapper.java:111)
	at org.neo4j.server.ServerBootstrapper.start(ServerBootstrapper.java:79)
	at org.neo4j.server.CommunityEntryPoint.main(CommunityEntryPoint.java:32)
Caused by: org.neo4j.kernel.lifecycle.LifecycleException: Component 'org.neo4j.server.database.LifecycleManagingDatabase@14e2e1c3' was successfully initialized, but failed to start. Please see the attached cause exception "org.neo4j.kernel.impl.core.EmbeddedProxySPI".
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:466)
	at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:107)
	at org.neo4j.server.AbstractNeoServer.start(AbstractNeoServer.java:212)
	... 3 more
Caused by: java.lang.RuntimeException: Error starting org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory, /Users/e/engines/neo4j/3.3.4/libexec/data/databases/graph.db
	at org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory.initFacade(GraphDatabaseFacadeFactory.java:211)
	at org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory.newFacade(GraphDatabaseFacadeFactory.java:126)
	at org.neo4j.server.CommunityNeoServer.lambda$static$0(CommunityNeoServer.java:58)
	at org.neo4j.server.database.LifecycleManagingDatabase.start(LifecycleManagingDatabase.java:88)
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:445)
	... 5 more
Caused by: org.neo4j.kernel.lifecycle.LifecycleException: Component 'com.graphaware.runtime.bootstrap.RuntimeKernelExtension@12f8b1d8' was successfully initialized, but failed to start. Please see the attached cause exception "org.neo4j.kernel.impl.core.EmbeddedProxySPI".
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:466)
	at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:107)
	at org.neo4j.kernel.extension.KernelExtensions.start(KernelExtensions.java:84)
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:445)
	at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:107)
	at org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory.initFacade(GraphDatabaseFacadeFactory.java:207)
	... 9 more
Caused by: java.lang.NoClassDefFoundError: org/neo4j/kernel/impl/core/EmbeddedProxySPI
	at com.graphaware.common.kv.GraphKeyValueStore.<init>(GraphKeyValueStore.java:31)
	at com.graphaware.runtime.metadata.GraphPropertiesMetadataRepository.<init>(GraphPropertiesMetadataRepository.java:57)
	at com.graphaware.runtime.GraphAwareRuntimeFactory.createRuntime(GraphAwareRuntimeFactory.java:63)
	at com.graphaware.runtime.bootstrap.RuntimeKernelExtension.start(RuntimeKernelExtension.java:111)
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:445)
	... 14 more
Caused by: java.lang.ClassNotFoundException: org.neo4j.kernel.impl.core.EmbeddedProxySPI
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 19 more
2018-05-24 13:04:40.570+0000 INFO  Neo4j Server shutdown initiated by request
```

**Comment by ikwattro** [comment]
You are using the wrong versions of the plugins. You use the ones for Neo4j 3.4.0 , 

take the 3.3.3.x versions of all the plugins

**Comment by smartinsightsfromdata** [comment]
Great!  
I've different versions of neo4j and I ended up with the wrong plugins.  Thanks
